Makefile

```diff
@@ -4,14 +4,14 @@ HELM_PLUGIN_DIR ?= $(HELM_HOME)/plugins/helm-2to3
 VERSION := $(shell sed -n -e 's/version:[ "]*\([^"]*\).*/\1/p' plugin.yaml)
 DIST := $(CURDIR)/_dist
 LDFLAGS := "-X main.version=${VERSION}"
-MOD_PROXY_URL ?= https://gocenter.io
+MOD_PROXY_URL ?= https://goproxy.io
```

Comment:
can we leave `https://gocenter.io` as GOPROXY as it has much better and bigger go modules database comparing to `goproxy.io`

Reply:
I've faced an issues with that proxy which were fixed after switching to that one. If it works for you I don't mind to change it back

Reply:
@rimusz There is a need to switch to `goproxy` as need `oras 0.7.0` for `helm v3.0.0-beta.x`. I am open to an alternative if you have one.